src/mscorlib/src/System/ThrowHelper.cs

```diff
@@ -570,6 +580,7 @@ internal enum ExceptionResource
         InvalidOperation_HandleIsNotInitialized,
         AsyncMethodBuilder_InstanceNotInitialized,
         ArgumentNull_SafeHandle,
+        MemoryDisposed,
```

Comment:
This is redundant with `Memory_ThrowIfDisposed` field

Reply:
I'll submit a PR to fix this. thanks for the catch